drakrun/drakrun/main.py

```diff
@@ -127,6 +127,81 @@ def wrapper(self: Karton, *args, **kwargs):
     return decorator
 
 
+class DrakrunConfig(Config):
+    """A typed wrapper for a Drakrun config. In the future, this will be handled
+    by Karton's typedconfig more directly
+    """
+
+    def __init__(self, config: Config) -> None:
+        super().__init__(config)
+
+    def get(self, option, fallback) -> Any:
+        return self.config.config.get("drakrun", option, fallback=fallback)
+
+    def getint(self, option, fallback) -> int:
+        return self.config.config.getint("drakrun", option, fallback=fallback)
+
+    def getboolean(self, option, fallback) -> bool:
+        return self.config.config.getboolean("drakrun", option, fallback=fallback)
+
+    @property
+    def out_interface(self) -> str:
+        return self.get("out_interface", "")
```

Comment:
I think I'd annotate the fallback arguments here and in the rest of the properties for better clarity, so:

```python
return self.get("out_interface", fallback="")
```